### ISSUE TITLE
Search button ay11

### ISSUE THREAD

**Issue by LouiaR** [comicrelief/component-library#280]
• This buttoninput element does not have a name available to an accessibility API. Valid names are: value undefined, aria-label undefined, aria-labelledby undefined.
